src/taskEvents/utils.js

```diff
@@ -1,11 +1,15 @@
 const BASE_URL = 'https://api.realdevsquad.com';
 
-function createElement({ type, attributes = {}, innerText }) {
+function createElement({ type, attributes = {}, innerText, innerHTML }) {
   const element = document.createElement(type);
   Object.keys(attributes).forEach((item) => {
     element.setAttribute(item, attributes[item]);
   });
-  element.textContent = innerText;
+  if (innerHTML) {
+    element.innerHTML = innerHTML;
+  } else if (innerText) {
+    element.textContent = innerText;
+  }
```

Comment:
as I only told you that we should not be using innerhtml so can we please remove this

Reply:
fixed